src/net.cpp

```diff
@@ -91,7 +91,6 @@ static constexpr auto FEELER_SLEEP_WINDOW{1s};
 /** Used to pass flags to the Bind() function */
 enum BindFlags {
     BF_NONE         = 0,
-    BF_EXPLICIT     = (1U << 0),
```

Comment:
Should the other enum entries be moved down so that there isn't a gap?

Reply:
These are stored in memory only, so it is irrelevant. They would have to stay the same if they were written on disk. The gap could cause somebody to ask themselves "why is there a gap" in the future.

There are already 3 ACKs, should I change it?

Reply:
Missed this, I think you should change it for clarity. I will ACK again.

Reply:
Changed, thanks!